internal/util/kafka.go

```diff
@@ -65,6 +63,83 @@ func (k kafkautil) NewWriter(topic string) *kafka.Writer {
 	}
 }
 
+func getSaslAndTLSConfig() (sasl.Mechanism, *tls.Config) {
+	username := config.DefaultConfig.KafkaUsername
+	password := config.DefaultConfig.KafkaPassword
+	saslmechanismName := config.DefaultConfig.KafkaSaslMechanism
+
+	tlsConfig, err := createTLSConfig(config.DefaultConfig.KafkaCAPath)
+	if err != nil {
+		fmt.Println("Error creating TLS configuration for Kafka:", err)
```

Comment:
config-manager logs to cloudwatch. Do you want to log these errors there? Or only to the stdout of the running process?

Reply:
Done!